docs/scenes/get-started/set-up-your-environment/SelectCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { ButtonBase, mergeClasses, useTheme } from '@expo/styleguide';
+import { useEffect, useState } from 'react';
+
+import { CALLOUT, HEADLINE } from '~/ui/components/Text';
+
+type Props = {
+  imgSrc: string;
+  darkImgSrc?: string;
+  alt: string;
+  title: string;
+  description?: string;
+  isSelected: boolean;
+  onClick: () => void;
+};
+
+export function SelectCard({
+  imgSrc,
+  darkImgSrc,
+  alt,
+  title,
+  description,
+  isSelected,
+  onClick,
+}: Props) {
+  const { themeName } = useTheme();
+  const [isDarkMode, setIsDarkMode] = useState(false);
+
+  useEffect(
+    function didMount() {
+      const darkMode =
+        themeName === 'dark' ||
+        (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches);
+      setIsDarkMode(darkMode);
+    },
+    [themeName]
+  );
```

Comment:
This code should be redundant, we have a global theme provider and management:
```jsx
import { useTheme } from '@expo/styleguide';
```

Reply:
This code also leads to always loading light mode images, no matter of the theme, which leads to flicker on load when user use the dark mode:

![xx](https://github.com/expo/expo/assets/719641/258f4f0d-4907-46eb-a880-4366dce823f3)

Reply:
I'll fix this, thanks for the suggestion!

Reply:
Oh right, my problem was that if you have "auto" set, then the `themeName` is not an accurate way to tell the theme, since then we need to listen for `prefers-color-scheme: dark`. I will remove that condition right now since it's causing that flicker but I can update our `useTheme()` hook later to support auto mode.